presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -536,7 +550,20 @@ void PrestoServer::initializeCoordinatorDiscoverer() {
   }
 }
 
-void PrestoServer::addAdditionalPeriodicTasks() {}
+void PrestoServer::addServerPeriodicTasks() {
+  periodicTaskManager_->addTask(
+      [server = this]() { server->populateMemAndCPUInfo(); },
+      1'000'000, // 1 second
+      "populate_mem_cpu_info");
+
+  const auto timeslice = SystemConfig::instance()->taskRunTimeSliceMicros();
+  if (timeslice > 0) {
+    periodicTaskManager_->addTask(
+        [server = this]() { server->yieldTasks(); }, timeslice, "yield_tasks");
+  }
+
+  addAdditionalPeriodicTasks();
```

Comment:
Sorry my fault. Can we remove this addServerPeriodicTasks and add the task outside. I noticed that addAdditionalPeriodicTasks are supposed to be virtually overridable. We don't want the system tasks to be overriden by extending classes. Thanks